docs/pages/auto-discovery/databases.mdx

```diff
@@ -1,183 +1,225 @@
+
+## Discovery Service configuration
+
+Here's an example database discovery configuration for the Discovery Service:
 
 ```yaml
-version: v3
-teleport:
-  join_params:
-    token_name: "/tmp/token"
-    method: token
-  proxy_server: "<Var name="teleport.example.com:443"/>"
-auth_service:
-  enabled: off
-proxy_service:
-  enabled: off
-ssh_service:
-  enabled: off
 discovery_service:
-  enabled: "yes"
-  discovery_group: "aws-prod"
-  aws:
-    # Database types. Valid options are:
-    # 'rds' - discovers AWS RDS and Aurora databases.
-    # 'rdsproxy' - discovers AWS RDS Proxy databases.
-    # 'redshift' - discovers AWS Redshift databases.
-    # 'redshift-serverless' - discovers AWS Redshift Serverless databases.
-    # 'elasticache' - discovers AWS ElastiCache Redis databases.
-    # 'memorydb' - discovers AWS MemoryDB Redis databases.
-    # 'opensearch' - discovers AWS OpenSearch Redis databases.
-  - types: ["rds"]
-    regions: ["us-east-1"]
-    tags:
-      "env": "prod" # Match database resource tags where tag:env=prod
-
-  - types: ["redshift", "redshift-serverless"]
-    regions: ["us-west-1"]
-    tags:
-      "env": "prod"
-    # Optional AWS role that the Discovery Service will assume to discover
-    # AWS-hosted databases. The IAM identity assigned to the host must be able
-    # to assume this role.
-    assume_role_arn: "arn:aws:iam::123456789012:role/example-role-name"
-    # Optional AWS external ID that the Database Service will use to assume
-    # a role in an external AWS account.
-    external_id: "example-external-id"
+    enabled: true
+    # discovery_group is used to group discovered resources into different
+    # sets. This is required when you have multiple Teleport Discovery services
+    # running. It prevents discovered services from colliding in Teleport when
+    # managing discovered resources.
+    # If two Discovery Services match the same resources, they must be in the
+    # same discovery group.
+    # If two Discovery Services match different resources, they must be in
+    # different discovery groups.
+    discovery_group: "disc-group"
+    # poll_interval is the cadence at which the discovery server will run each of its
+    # discovery cycles. The default is 5m.
+    poll_interval: 5m
+    # Matchers for discovering AWS-hosted resources.
+    aws:
+      # AWS resource types. Valid database discovery types are:
+      # 'rds' - discovers and registers Amazon RDS and Aurora databases.
+      # 'rdsproxy' - discovers and registers Amazon RDS Proxy databases.
+      # 'redshift' - discovers and registers Amazon Redshift databases.
+      # 'redshift-serverless' - discovers and registers Amazon Redshift Serverless databases.
+      # 'elasticache' - discovers and registers Amazon ElastiCache Redis databases.
+      # 'memorydb' - discovers and registers Amazon MemoryDB Redis databases.
+      # 'opensearch' - discovers and registers Amazon OpenSearch Redis databases.
```

Comment:
I think it would make sense to move "discovers and registers" to the comment above the list of discovery types to avoid repetition

Reply:
done